src/plot_api/helpers.js

```diff
@@ -185,6 +213,11 @@ exports.cleanLayout = function(layout) {
     // supported, but new tinycolor does not because they're not valid css
     Color.clean(layout);
 
+    // clean the layout container in layout.template
+    if(layout.template && layout.template.layout) {
+        exports.cleanLayout(layout.template.layout);
```

Comment:
Good call here. I guess this means templates with `layout.xaxis1` didn't work previously, correct?